.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: scala
+
+scala:
+  - 2.12.10
+  - 2.13.1
+jdk:
+  - openjdk8
+  - openjdk11
+
+install:
+  - ./gradlew --version
+  - mvn --version
```

Comment:
Here and below we should call `demo/maven/mvnw` instead of `mvn`.

Reply:
Ok, didn't notice that was there.  I'll give it a try.